packages/headless/src/controllers/field-suggestions/facet/headless-field-suggestions.ts

```diff
@@ -20,13 +19,13 @@ import {
   SearchSection,
 } from '../../../state/state-sections';
 import {loadReducerError} from '../../../utils/errors';
-import {omit} from '../../../utils/utils';
 import {
   buildController,
   Subscribable,
 } from '../../controller/headless-controller';
 import {determineFacetId} from '../../core/facets/_common/facet-id-determinor';
 import {buildFacetSearch} from '../../core/facets/facet-search/specific/headless-facet-search';
+import {FacetOptions} from '../../facets/facet/headless-facet-options';
```

Comment:
As I was working on the migration docs for the fieldSuggestionOptions removal, I noticed that the `FacetOptions` annotations don't appear in the parsed_doc.json that we use to generate docs.
So we don't have anything about that currently [in the reference docs](https://docs.coveo.com/en/headless/latest/reference/search/controllers/facet/#facetpropsfacetoptions).
That's a bit annoying, because it means that by removing the FieldsSuggestionFacetSearchOptions, we also lose their docs.

It seems fine [for category facets](https://docs.coveo.com/en/headless/latest/reference/search/controllers/category-facet/#categoryfacetoptions), somehow. 

I admit this is a bit mysterious to me, but if there's a simple fix I'm not seeing it would be cool.
Otherwise I can create a separate Jira for later?

Reply:
👍 
Created https://coveord.atlassian.net/browse/KIT-2174